packages/components/timed-text-editor/WrapperBlock.js

```diff
@@ -164,42 +192,65 @@ class WrapperBlock extends React.Component {
     }
   };
 
+  generatePreviousTimes = (data) => {
+    if(data){
+      let prevTimes = '';
+
+      for (let i = 0; i < data.length; i++) {
+          prevTimes += `${ parseInt(data[i].start) } `;
+      }
+  
+      return prevTimes;
+    }
+    else{
+      return '';
+    }
+  }
+
   render() {
     let startTimecode = this.state.start;
     if (this.props.blockProps.timecodeOffset) {
       startTimecode += this.props.blockProps.timecodeOffset;
     }
-
     const speakerElement = (
       <SpeakerLabel
         name={ this.state.speaker }
         handleOnClickEdit={ this.handleOnClickEdit }
         isEditable={this.props.blockProps.isEditable}
+        start={startTimecode}
       />
     );
 
     const timecodeElement = (
       <span className={ style.time } onClick={ this.handleTimecodeClick }>
-        {shortTimecode(startTimecode)}
+        {startTimecode? shortTimecode(startTimecode): 'NA'}
       </span>
     );
 
     return (
-      <div className={ style.WrapperBlock }>
+      <div
+      className={ this.props.blockProps.showTimecodes ? style.WrapperBlock: style.WrapperBlockNoTimecodes }>
         <div
-          className={ [ style.markers, style.unselectable ].join(' ') }
+          className={ [ 
+            this.props.blockProps.showTimecodes ? style.markers: style.markersNoTimecodes
+            ,
+            style.unselectable ].join(' ') }
           contentEditable={ false }
         >
           {this.props.blockProps.showSpeakers ? speakerElement : ''}
-
           {this.props.blockProps.showTimecodes ? timecodeElement : ''}
         </div>
-        <div className={ style.text }>
+        <div  
+          data-start={this.state.start} 
+          data-start-srcoll-sync={parseInt(this.state.start)} 
```

Comment:
```suggestion
          data-start-scroll-sync={parseInt(this.state.start)} 
```
If I'm not mistaken :D

Reply:
yeah, you'd want to check if it's a typo there or also in other part of the code, eg where ti does `document.querySelector` etc..